conf/certificate/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Do we need this file?  Shouldn't the `.gitignore` in the root directory handle this?

Reply:
Yes, we need this **empty** directory to be created
https://git.wiki.kernel.org/index.php/GitFaq#Can_I_add_empty_directories.3F